packages/flame/lib/src/game/base_game.dart

```diff
@@ -240,4 +240,10 @@ class BaseGame extends Game with FPSCounter {
     return DateTime.now().microsecondsSinceEpoch.toDouble() /
         Duration.microsecondsPerSecond;
   }
+
+  @override
+  Vector2 projectCoordinate(Offset value) {
```

Comment:
Should this be directly on `BaseGame`? Isn't this more suitable on the `Camera`, or somewhere else?

Reply:
Ah, I get the thinking now after reading the doc on `Game`. Not completely convinced though.

Reply:
This is a method from Game actually, which we override to be able to add the BaseGame's transforms into the event system, which lives on the Game class.

Needs to be this way, otherwise we will need to hook the Camera directly on the Game class.

Reply:
This will be really good for forge2d too I actually, since it has another type of coordinate system for the world.